src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -86,49 +97,51 @@ export default function ConfirmedAppointmentListItem({
             </dl>
           )}
         </div>
-        {hasInstructions(appointment) && (
-          <div className="vads-u-flex--1 vads-u-margin-top--2 vaos-u-word-break--break-word">
-            <dl className="vads-u-margin--0">
-              <dt className="vads-u-font-weight--bold">
-                {getAppointmentInstructionsHeader(appointment)}
-              </dt>
-              <dd>{getAppointmentInstructions(appointment)}</dd>
-            </dl>
-          </div>
-        )}
+        {hasInstructions(appointment) &&
+          !isPastAppointment && (
+            <div className="vads-u-flex--1 vads-u-margin-top--2 vaos-u-word-break--break-word">
+              <dl className="vads-u-margin--0">
+                <dt className="vads-u-font-weight--bold">
+                  {getAppointmentInstructionsHeader(appointment)}
+                </dt>
+                <dd>{getAppointmentInstructions(appointment)}</dd>
+              </dl>
+            </div>
+          )}
       </div>
 
-      {!canceled && (
-        <div className="vads-u-margin-top--2">
-          <AddToCalendar
-            summary={getAppointmentTypeHeader(appointment)}
-            description={
-              hasInstructions(appointment)
-                ? `${getAppointmentInstructionsHeader(
-                    appointment,
-                  )}. ${getAppointmentInstructions(appointment)}`
-                : ''
-            }
-            location={getAppointmentAddress(appointment, facility)}
-            duration={getAppointmentDuration(appointment)}
-            startDateTime={getMomentConfirmedDate(appointment).toDate()}
-            endDateTime={getMomentConfirmedDate(appointment)}
-          />
-          {showCancelButton && (
-            <button
-              onClick={() => cancelAppointment(appointment)}
-              aria-label="Cancel appointment"
-              className="vaos-appts__cancel-btn va-button-link vads-u-margin--0 vads-u-flex--0"
-            >
-              Cancel appointment
-              <span className="sr-only">
-                {' '}
-                on {getAppointmentDate(appointment)}
-              </span>
-            </button>
-          )}
-        </div>
-      )}
+      {!canceled &&
+        !isPastAppointment && (
```

Comment:
I think we also need to hide the video connect location info for past appointments.